pandas/core/series.py

```diff
@@ -4012,6 +4012,13 @@ def _reduce(
         If we have an ndarray as a value, then simply perform the operation,
         otherwise delegate to the object.
         """
+        if name == "var" and is_timedelta64_dtype(self.dtype):
+            raise TypeError(
```

Comment:
is there anyway you can do this in nanops instead? it is very awkward to do here

Reply:
Looks like this is no longer necessary, removed.